Views/CreateSeedView.swift

```diff
@@ -31,7 +31,7 @@ class CreateSeedView: UIView, UITextFieldDelegate {
 
   lazy var specieButton: UIButton = {
     let specieButton = UIButton(frame: CGRect.zero)
-    specieButton.setTitle("Avoine", for: .normal)
+    specieButton.setTitle("Abricotier", for: .normal)
```

Comment:
What is the use of setting this to "Abricotier" ?

Reply:
It's the first specie in alphabetical order. It's in French but we will use locales soon.